cobc/parser.y

```diff
@@ -14488,6 +14512,8 @@ exit_body:
 		}
 		current_statement->statement = STMT_EXIT_SECTION;
 		cb_emit_goto (CB_LIST_INIT (current_section->exit_label), NULL);
+		// emit_prof_call (COB_PROF_EXIT_PARAGRAPH);
```

Comment:
Actually, it wouldn't work, because the `begin_statement` is at the `ENTRY ` keyword, so any `emit_statement` afterwards will be added after it. Finally, I ended up emitting `cob_prof_goto` before the `goto`, which is a weaker form of `cob_prof_exit_paragraph` without the paragraph name.